### DISCUSSION TITLE
Motion detection settings

### DISCUSSION THREAD

**Discussion by raphthibert** [jmcollin78/versatile_thermostat#505] [Q&A]
Hello, 

   I have trouble setting up the motion detection (I would like to set it up only in one room, my office where I work sometimes).
   The idea is the get the comfort temperature during the day when I am in the room ( I have a motion detector).
   I am also using scheduler.
   
   When I activate the motion detection function, I think it gives the eco setting when no motion detected and the confort value when motion detected (but from the TRV itself, not settings from scheduler). I would like to follow the scheduler setting and only when motion detected get the confort setting if eco at that time, is it possible?
   
   Thanks in advance for the help!

**Comment by jmcollin78** [comment]
Hello,

To use the motion detection, you have to set the VTherm to the 'Activity' preset. If no motion is detected, VTherm sends the Eco temperature preset value (or what you have configured) and if motion is detected the Comfort value (or you have configured).

> I would like to follow the scheduler setting and only when motion detected get the confort setting if eco at that time

I'm not sure to understand but anyway, this is not possible to combine with Scheduler out of the box. If schedule change the preset, it will remove the Activity preset and then the VTherm is no more responding to motion.

--> You have to choose: do you want to control the temperature with the motion or with a Scheduler. If Scheduler says 'Comfort' and there is no motion. What preset do you want ? If Comfort, then you just don't need motion, if Eco then you just don't need scheduler because the Activity preset do it for you.

I see other possibilities:
1. do a specific automation,
2. use Scheduler to program the Activity preset so that the Vtherm will be controlled by motion only at certain time.
